python/tchannel/tornado/dispatch.py

```diff
@@ -88,17 +93,25 @@ def handle_call(self, request, connection):
                 request.id,
             )
         else:
-            response = Response(id=request.id, connection=connection)
-            yield self._call_endpoint(endpoint, request, response)
-
-    def route(self, rule, **opts):
+            response = Response(id=request.id,
+                                tracing=request.tracing,
+                                connection=connection)
+
+            from .tchannel import TChannelProxy
```

Comment:
I looked at the implementation. There's no reason it couldn't be moved to this file. Especially because this is the only place it is consumed. 

There's no hard and fast rule dictating that classes need to be in a file named similar to their class name.